core/discord_bot.py

```diff
@@ -169,42 +171,33 @@ async def _process_invites(self):
         print("Discord > Invite processor has been stopped.")
 
     async def _send_message(self, *args, **kwargs) -> Union[discord.Message, discord.WebhookMessage, None]:
+        self.name = DiscordConfig.serverName if DiscordConfig.serverName is not "" else "Bridge Bot"
```

Comment:
This should be overriden by the `username` kwarg

Reply:
Added webhook condition